map_gen/maps/danger_ores/modules/restart_command.lua

```diff
@@ -61,7 +62,7 @@ return function(config)
         end
 
         local statistics = {
-            scenario = config.scenario_name,
+            scenario = ScenarioInfo.get_map_name(),
```

Comment:
```suggestion
            scenario = config.scenario_name or 'Danger ore',
```
It'd be consistent with the names that get used for the next map.

Reply:
Wasn't sure if `statistics.scenario_name` was used.
isn't `config.scenario_name` the next map?
and `ScenarioInfo.get_map_name()` is formatted for display, so it's looks better in discord.
could add both to the statistics array if needed when sent to the server

Reply:
> Wasn't sure if `statistics.scenario_name` was used.

There is a key/value store on the server, which is what `Server.set_data('danger_ores_data', tostring(end_epoch), statistics)` is doing. The data isn't needed, but is just collected if anyone (Jayefuu) what's to look at them.

> isn't config.scenario_name the next map?

It's the map that is set in the presets e.g. here <https://github.com/Refactorio/RedMew/blob/develop/map_gen/maps/danger_ores/presets/danger_ore_deadlock_beltboxes_ore_only.lua#L119>. So assuming the presets are correct it would be the current map. By default the maps restart to themselves unless an admin changes it with `/config-restart`. 
```lua
        local start_game_data = Restart.get_start_game_data()
        local new_map_name = start_game_data.name
```
On line 36 determines the next map.

> and ScenarioInfo.get_map_name() is formatted for display, so it's looks better in discord.

There are pros and cons to using `ScenarioInfo.get_map_name()`. My intital thought is it would be inconsistent with the name of the map we show for the next map in map-promotion. e.g:
![image](https://user-images.githubusercontent.com/10003572/169603468-1acc3235-38ac-415b-99b5-d1b5b193f973.png). Technically admins can also change what `ScenarioInfo.get_map_name()` returns by editing the textbox in the map info gui (for admins the fields are editable).

> could add both to the statistics array if needed when sent to the server

I'd probably stick with sending `config.scenario_name` as it's what was being sent before.

Reply:
Should I move it out of the title then? Doesn't look good in title when it's dashed.

![image](https://user-images.githubusercontent.com/762387/169610487-bffa4b26-6647-4e99-8b2d-b5eccae0a100.png)
![image](https://user-images.githubusercontent.com/762387/169610713-eeb40d23-e2b4-4776-8ebb-7dd2d5f53bfe.png)
![image](https://user-images.githubusercontent.com/762387/169610898-adba47f8-79a9-47c3-8960-c6a1af635849.png)

Reply:
I think it's fine in the title.